hyperdome_client/hyperdome_client.py

```diff
@@ -208,6 +204,13 @@ def send_message(self):
         Send the contents of the message box to the server to be forwarded to
         either counsel or guest.
         """
+        if self.uid and not self.therapist:
+            self.therapist = self.session.post(
+                f"{self.server.url}/request_therapist",
```

Comment:
this seems like it should happen somewhere else? User should be connected to someone before sending a message.